src/components/RefinementList/RefinementListItem.js

```diff
@@ -45,11 +45,6 @@ RefinementListItem.propTypes = {
   templateProps: PropTypes.object.isRequired,
   cssClasses: PropTypes.shape({
     item: PropTypes.string,
-    selectedItem: PropTypes.string,
-    label: PropTypes.string,
-    checkbox: PropTypes.string,
-    labelText: PropTypes.string,
-    count: PropTypes.string,
```

Comment:
For this I was more talking about why do we have switched from string to object? Since there is only one element rendered in the component it makes sense to have `className` rather than `cssClasses`.